feature/profile/src/commonMain/kotlin/io/github/confsched/profile/ProfileEditScreen.kt

```diff
@@ -403,6 +403,7 @@ private fun FormField<String>.InputField(
             onValueChange = { onValueChange(it) },
             isError = hasError,
             maxLines = 1,
+            singleLine = true,
```

Comment:
[nitpick] Since singleLine = true already implies maxLines = 1, we can safely omit the maxLines parameter. 👍

<img width="664" height="500" alt="image" src="https://github.com/user-attachments/assets/fbad1aa0-8779-43b9-a30a-0305209f4f90" />